diameter-stack/build.gradle

```diff
@@ -2,8 +2,77 @@ plugins {
   id "org.jetbrains.kotlin.jvm" version "1.2.30"
   id "idea"
   id "java-library"
+  id "signing"
+  id "maven"
 }
 
+version="1.0.0-SNAPSHOT"
+
+task javadocJar(type: Jar) {
+    classifier = 'javadoc'
+    from javadoc
+}
+
+task sourcesJar(type: Jar) {
+    classifier = 'sources'
+    from sourceSets.main.allSource
+}
+
+artifacts {
+    archives javadocJar, sourcesJar
+}
+
+signing {
+    sign configurations.archives
+}
+
+
+uploadArchives {
+  repositories {
+    mavenDeployer {
+      beforeDeployment { MavenDeployment deployment -> signing.signPom(deployment) }
```

Comment:
IDE is complaining about 'MavenDeployment'.  Do I need to do something to fix this?